src/service/VotingService.ts

```diff
@@ -54,13 +53,11 @@ export class VotingService {
                 logger.info(`Voting File for node ${nodePreset.name} has been previously generated. Reusing...`);
                 return;
             }
+            const votingPrivateKey = nodeAccount?.voting.privateKey;
+            if (!votingPrivateKey) {
+                throw new Error('Voting key should have been previously generated!!!');
```

Comment:
Can we tell the user what to do in this case?

Reply:
Updated, atm they need to hard reset the target folder using the original custom preset. It should be an odd case where the voting key file got lost